docs/proposals/secure-addon-installation/secure-addon-installation.md

```diff
@@ -0,0 +1,154 @@
+    ```bash
+   kubectl set image deployment/service-catalog-catalog-webhook svr=mszostok/service-catalog-amd64:fixed-userinfo
+    ```
+3. Check out the [`impersonate-helm-broker`](https://github.com/mszostok/impersonate-helm-broker) repository locally.
+4. Install the impersonate-helm-broker. Run:
+    ```bash
+    kubectl create -f ./deploy
+    ```
+5. Create a ServiceInstance as the admin who has all privileges. Run: 
+    ```bash
+   kubectl create -f ./example/si_admin.yaml --as=admin@kyma.cx
+    ```
+6. Check the status of the created ServiceInstance. Run:
+    ```bash
+    kubectl get serviceinstance redis-admin
+    ```   
+   The ServiceInstance should be in the **Ready** state. See the example of an output:
+   ```bash
+   NAME             CLASS                       PLAN      STATUS   AGE
+   redis-admin      ClusterServiceClass/redis   default   Ready    10s
+   ```
+
+7. Create a ServiceInstance as **user2** that doesn't have permission for creating Kubernetes resources other than ServiceInstances. Run: 
+    ```bash
+   kubectl create -f ./example/si_user2.yaml --as=user2@kyma.cx
+    ```
+8. Check the status of the created ServiceInstance. Run:
+    ```bash
+    kubectl get serviceinstance redis-user2
+    ```   
+   The ServiceInstance should be in the **OrphanMitigationSuccessful** state. See the example of an output:
+   ```bash
+   NAME          CLASS                       PLAN      STATUS                       AGE
+   redis-user2   ClusterServiceClass/redis   default   OrphanMitigationSuccessful   13m
+   ```
+
+   You can check the Service Catalog controller logs to find the provisioning error. To do so, run:
+   ```bash
+    kubectl logs $(kubectl get po -n kyma-system -l app=service-catalog-catalog-controller-manager --no-headers -ocustom-columns=name:.metadata.name) -n kyma-system -c controller-manager | grep "Error syncing ServiceInstance impersonate-helm-broker/redis-user2"
+    ```
+   This is the expected output: 
+   ```bash
+    I0615 19:07:13.243344       1 controller.go:404] Error syncing ServiceInstance impersonate-helm-broker/redis-user2 (retry: 0/15): Error provisioning ServiceInstance of ClusterServiceClass (K8S: "123-123-123-123-123-123" ExternalName: "redis") at ClusterServiceBroker "impersonate-helm-broker": Status: 500; ErrorMessage: <nil>; Description: namespaces is forbidden: User "user2@kyma.cx" cannot create resource "namespaces" in API group "" at the cluster scope; ResponseError: <nil>
+   ```
+
+### Follow-ups
+
+The following follow-ups need to be executed:
+
+1. The **userInfo** field is always updated with the Service Catalog controller ServiceAccount. 
+   
+   Accepted solution: [This PR](https://github.com/kubernetes-sigs/service-catalog/pull/2822) needs to be merged to fix the user's identity feature. Check the PR for more details.
+
+2. Console Backend Service creates a ServiceInstance and ServiceBindings using controller ServiceAccount instead of a user account who triggered that call. 
```

Comment:
```suggestion
2. Console Backend Service creates a ServiceInstance and ServiceBindings using controller ServiceAccount instead of a user account that triggered that call. 
```